CentralComputing/Utils.cpp

```diff
@@ -41,7 +41,7 @@ void Utils::print(LogLevel level, const char * format, ...){
       }
 }
 
-int Utils::toggle_GPIO(int GPIONumber, bool switchVal) {
+int Utils::set_GPIO(int GPIONumber, bool switchVal) {
```

Comment:
Change to `bool`? since the only values will be 1 or 0, it makes sense to change it to a `bool`

Reply:
(the return value)